app/policies/user_policy.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class UserPolicy < ApplicationPolicy
```

Comment:
From this class need to remove policy methods

show? - Validate with JWT
create? - that's free method for all
update?  - Validate with JWT

Reply:
check it